ReactAndroid/src/main/java/com/facebook/react/views/swiperefresh/SwipeRefreshLayoutManager.java

```diff
@@ -82,6 +83,21 @@ public void run() {
     });
   }
 
+  @ReactProp(name = "progressViewOffset", defaultFloat = 0)
+  public void setProgressViewOffset(final ReactSwipeRefreshLayout view, final float offset) {
+    // Use `post` to get progress circle diameter properly
+    // Otherwise returns 0
+    view.post(new Runnable() {
+      @Override
+      public void run() {
+        int diameter = view.getProgressCircleDiameter();
+        int start = Math.round(PixelUtil.toPixelFromDIP(offset)) - diameter;
+        int end = Math.round(PixelUtil.toPixelFromDIP(offset+48));
```

Comment:
Is there a better way to get that 48? If not maybe put it in a constant like `REFRESH_TRIGGER_DISTANCE` so it's more clear what it is.

Also nit: spaces before and after `+` and after `,`.